src/models/expenseModel.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-console */
+/* eslint-disable function-paren-newline */
+/* eslint-disable no-shadow */
+'use strict';
+
+const expenses = [];
+
+const expenseModel = {
```

Comment:
expenseModel  could be a Class, but that's ok

Reply:
agree